schema.json

```diff
@@ -127,6 +127,9 @@
 				},
 				"enableHelm": {
 					"type": "boolean"
+				},
+				"loadRestrictor": {
+					"type": "string"
```

Comment:
As far as I can tell, `LoadRestrictionsNone` and `LoadRestrictionsRootOnly` are the only two values supported by the `--load-restrictor` flag.

We might want to use a custom type here with only those two values enumerated as acceptable -- the benefit being that an invalid value will be caught up front when config is loaded instead of later during the rendering attempt.